gearcmd/worker.go

```diff
@@ -40,11 +40,17 @@ func (conf TaskConfig) Process(job baseworker.Job) ([]byte, error) {
 	}
 	if err != nil {
 		data["error_message"] = err.Error()
-		log.Printf(kayvee.FormatLog("gearcmd", "error", "END_WITH_ERROR", data))
+		data["type"] = "gauge"
+		data["value"] = 0
+		data["success"] = false
```

Comment:
As I understand it, even adding a new line would require modifying the toml file. Currently, the data.toml file is empty. https://github.com/Clever/salt/pull/582 adds a LibratoOutput to data.toml which will parse the output and send to Librato if it matches this specific event.